.github/workflows/main-ci.yml

```diff
@@ -12,11 +12,15 @@ jobs:
           profile: minimal
           toolchain: nightly
           override: true
-          components: rustfmt
+          components: rustfmt, clippy
       - uses: actions-rs/cargo@v1
         with:
           command: fmt
           args: --all -- --check
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
whats the token for?

Reply:
It adds comments onto the PR directly and it needs the token for that.
ref: (link)[https://github.com/actions-rs/clippy-check#limitations]

Reply:
I'm not sure it actually needs it though, it looks like `clippy-check` has not been developed in a long time and we can just use the `cargo` action:
https://github.com/actions-rs/meta/blob/master/recipes/quickstart.md

The existing CI manages to show the warnings in the PR without any such token

Reply:
Ah your right, had not noticed that. Will change